nwb_conversion_tools/basedatainterface.py

```diff
@@ -25,8 +25,8 @@ def get_metadata(self):
     @classmethod
     @abstractmethod
     def get_conversion_options_schema(cls):
-        return get_schema_from_method_signature(cls.convert_data, exclude=['nwbfile', 'metadata_dict'])
+        return get_schema_from_method_signature(cls.convert_data, exclude=['nwbfile', 'metadata_dict', 'stub_test'])
```

Comment:
why is stub_test removed?

Reply:
Trying to test at the moment if it can or can't be, but just about all interfaces expect it. It's mostly as you use this in an actual conversion framework (debugging with tank-lab atm), passing the conversion_options dictionary in the right way, you would have to repeat stub_test on a lot of the interfaces if it isn't handled globally.